db/migrate/20240626094521_add_shipping_address_to_customers.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class AddShippingAddressToCustomers < ActiveRecord::Migration[7.1]
+  change_table :customers, bulk: true do |t|
```

Comment:
Could customers have more than one shipping address?

Is there any chance a customer change their address in the middle of an invoice period or something and the taxes are not properly applied?

Reply:
Based on current requirements, customer can have only one shipping address.

Sure, if shipping address got updated, there is a chance that fetched taxes would also changed. However, each organization is responsible for it and I am not sure we should have any extra validation here